Modules/Registration/Metricsv4/test/itkMattesMutualInformationImageToImageMetricv4Test.cxx

```diff
@@ -122,6 +125,7 @@ TestMattesMetricWithAffineTransform(TInterpolator * const interpolator, const bo
       ++ri;
     }
   }
+  if (initializeFixedImage)
```

Comment:
ITK does not initialize the image buffer by default. That is the values are undefined and access is a valgrind defect. If the values are not going to be explicitly set then `imgFixed->Allocate(true);` should be executed so that the buffer is initialized to zero.

Reply:
Updated so that the fixedImage is always initialized `imgFixed->Allocate(true);`